src/Nethermind/Nethermind.Trie.Test/RustVerkleLibTest.cs

```diff
@@ -75,21 +76,60 @@ public class RustVerkleLibTest
 
 
         [Test]
-        public void TestInsertGet()
+        public void TestInsertByteGet()
         {
-            byte[] one = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1};
-            byte[] one32 = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
+            byte[] one = 
+            {
+                0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1
+            };
+            byte[] one32 = 
+            {
+                1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
+            };
+
+            IntPtr trie = RustVerkleLib.VerkleTrieNew();
+
+            RustVerkleLib.VerkleTrieInsert(trie, one, one32);
+            RustVerkleLib.VerkleTrieInsert(trie, one32, one);
+
+            RustVerkleLib.VerkleTrieGet(trie, one32).Should().BeEquivalentTo(one);
+            RustVerkleLib.VerkleTrieGet(trie, one).Should().BeEquivalentTo(one32);
+        }
+        
+        [Test]
+        public void TestInsertSpanGetSpan()
+        {
+            Span<byte> one = new byte[]{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1};
+            Span<byte> one32 = new byte[]{1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
+
+            IntPtr trie = RustVerkleLib.VerkleTrieNew();
+
+            RustVerkleLib.VerkleTrieInsert(trie, one, one32);
+            RustVerkleLib.VerkleTrieInsert(trie, one32, one);
+
+            RustVerkleLib.VerkleTrieGetSpan(trie, one32).ToArray().Should().BeEquivalentTo(one.ToArray());
```

Comment:
Lets add extension methods like: 

public static GenericCollectionAssertions<T> Should<T>(this ReadOnlySpan<T> value)
    => value.ToArray().Should();

public static StringAssertions Should<T>(this Span<char> value)
    => value.ToString().Should();

And same for BeEquivalentTo